packages/web/modals/validator-squad.tsx

```diff
@@ -194,14 +244,31 @@ const ValidatorSquadContent: FunctionComponent<ValidatorSquadContentProps> =
       getSortedRowModel: getSortedRowModel(),
     });
 
-    const handleSearchInput = () => console.log("search");
+    const { rows } = table.getRowModel();
+
+    const tableContainerRef = useRef<HTMLDivElement>(null);
+
+    const rowVirtualizer = useVirtual({
```

Comment:
Did we have virtualization on any of our other tables? Just want to make sure we don't have an existing solution we can use. Otherwise `react-virtual` is a good solution

Reply:
We've gotten the virtual table to work on the pools page, so we shouldn't need to import additional dependencies. Did you take a look there? Like the paginated table component?

Reply:
You can take a look at the previews as well to see if it works there

https://github.com/osmosis-labs/osmosis-frontend/pull/1756#issuecomment-1622120012